src/main/java/org/folio/dao/transactions/BatchTransactionDAO.java

```diff
@@ -8,6 +8,8 @@
 import java.util.List;
 
 public interface BatchTransactionDAO {
+  String TRANSACTIONS_TABLE = "transaction";
```

Comment:
I saw many places it is final static and can we put all of them in one place and used for other places as well
![image](https://github.com/folio-org/mod-finance-storage/assets/113523904/f6843145-cdee-4988-82c6-09ecd428c126)

Reply:
EncumbranceDAO was deleted. I am going to use a reference in BudgetService.